rasterio/crs.py

```diff
@@ -321,6 +333,25 @@ def from_epsg(cls, code):
         obj._crs = _CRS.from_epsg(code)
         return obj
 
+    @classmethod
+    def from_authority(cls, auth_name, code):
+        """Make a CRS from an authority name and authority code
+
+        .. versionadded:: 1.1.7
```

Comment:
If you would prefer this to be a 1.2 thing, just let me know and I can update this PR and point to master.

Reply:
No, this is fine. This is more of a "missing feature is a bug".

Reply:
Makes sense.